DevProxy.Abstractions/Utils/MSGraphDbUtils.cs

```diff
@@ -48,61 +49,64 @@ public static async Task<int> GenerateMSGraphDbAsync(ILogger logger, bool skipIf
             var modifiedToday = dbFileInfo.Exists && dbFileInfo.LastWriteTime.Date == DateTime.Now.Date;
             if (modifiedToday && skipIfUpdatedToday)
             {
-                logger.LogInformation("Microsoft Graph database already updated today");
+                _logger.LogInformation("Microsoft Graph database already updated today");
                 return 1;
             }
 
-            await UpdateOpenAPIGraphFilesIfNecessaryAsync(appFolder, logger, cancellationToken);
-            await LoadOpenAPIFilesAsync(appFolder, logger, cancellationToken);
+            await UpdateOpenAPIGraphFilesIfNecessaryAsync(appFolder, cancellationToken);
+            await LoadOpenAPIFilesAsync(appFolder, cancellationToken);
             if (_openApiDocuments.Count < 1)
             {
-                logger.LogDebug("No OpenAPI files found or couldn't load them");
+                _logger.LogDebug("No OpenAPI files found or couldn't load them");
                 return 1;
             }
 
-            var dbConnection = MSGraphDbConnection;
-            await CreateDbAsync(dbConnection, logger, cancellationToken);
-            await FillDataAsync(dbConnection, logger, cancellationToken);
+            await CreateDbAsync(cancellationToken);
+            await FillDataAsync(cancellationToken);
 
-            logger.LogInformation("Microsoft Graph database successfully updated");
+            _logger.LogInformation("Microsoft Graph database successfully updated");
 
             return 0;
         }
         catch (Exception ex)
         {
-            logger.LogError(ex, "Error generating Microsoft Graph database");
+            _logger.LogError(ex, "Error generating Microsoft Graph database");
             return 1;
         }
 
     }
 
     private static string GetGraphOpenApiYamlFileName(string version) => $"graph-{version.Replace(".", "_", StringComparison.OrdinalIgnoreCase)}-openapi.yaml";
 
-    private static async Task CreateDbAsync(SqliteConnection dbConnection, ILogger logger, CancellationToken cancellationToken)
+    private async Task CreateDbAsync(CancellationToken cancellationToken)
     {
-        logger.LogInformation("Creating database...");
+        _logger.LogInformation("Creating database...");
+
+        var dbConnection = MSGraphDbConnection;

```

Comment:
I don't think we need this assignment anymore since we already have it in a class member

Reply:
fixed